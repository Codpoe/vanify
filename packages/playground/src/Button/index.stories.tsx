import { Meta, StoryObj } from '@storybook/react';
import { Button } from '.';

const meta: Meta<typeof Button> = {
  title: 'Cool/So/Hello/Button',
  component: Button,
  tags: ['autodocs'],
};

export default meta;

export const Primary: StoryObj<typeof Button> = {
  args: {
    type: 'primary',
  },
};

export const Secondary: StoryObj<typeof Button> = {
  args: {
    type: 'secondary',
  },
};
