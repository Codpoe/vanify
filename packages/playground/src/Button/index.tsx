import React from 'react';
import { Tag } from '../Tag';

export const Button: React.FC = () => {
  return (
    <button className="btn">
      Hello Button
      <Tag />
    </button>
  );
};

export default Button;
