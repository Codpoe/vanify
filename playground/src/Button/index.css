.btn {
  height: 32px;
  padding: 8px 16px;
  outline: none;
  background-color: #ff0050;
  border: 1px solid #ff0050;
  border-radius: 2px;

  &:hover {
    background-color: #db004c;
    border-color: #db004c;
  }
}
