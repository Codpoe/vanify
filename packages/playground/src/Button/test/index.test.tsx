import React from 'react';
import { render } from '@testing-library/react';
import Button from '..';

test('render and match snapshot', () => {
  const result = render(<Button />);
  expect(result.container).toMatchSnapshot();
});
