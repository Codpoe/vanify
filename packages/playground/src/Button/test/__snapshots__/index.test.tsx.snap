// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render and match snapshot 1`] = `
<div>
  <button
    class="btn"
  >
    Hello Button
    <span
      class="tag"
    >
      Hello Tag
    </span>
  </button>
</div>
`;
